### PR TITLE
RHIDP-1037-RN: Adding cross-ref in the new features section for more info

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve this issue: https://issues.redhat.com/browse/RHIDP-1710 [mention that change in the GitHub Organization catalog provider is a breaking change and add cross-references in the new features section for more details]

### CODE REVIEW

File: release-notes/con-relnotes-notable-features.adoc [hunk 1]
```diff
@@ -31,6 +31,8 @@ For more information, see the link:{LinkAdminGuide}#con-rbac-overview_admin-rhdh
 
 The GitHub Organization catalog module for the new backend system switched to a new `GithubMultiOrgEntityProvider` provider, which allows ingesting data from multiple GitHub organizations. This new provider brings in changes to the default plugin configuration settings.
```
Comment (kim-tsao) [inline release-notes/con-relnotes-notable-features.adoc:32]: Can we call out that this is a breaking change that customers will need to react to?  I think we may need to have a migration instructions

Comment (shipsing) [reply release-notes/con-relnotes-notable-features.adoc]: I have added a cross-ref to the procedure that covers the upgrade and configuration part: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html-single/getting_started_with_red_hat_developer_hub/index#enabling-github-organization-member-discovery-in-red-hat-developer-hub

File: release-notes/con-breaking-changes.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
+[id='con-breaking-changes_{context}']
+= Breaking changes
+
+This section lists breaking changes with {product} {product-version}:
+
+== Configuration change in the GitHub Organization catalog provider  
+
+The GitHub Organization catalog module for the new backend system switched to a new `GithubMultiOrgEntityProvider` provider, which allows ingesting data from multiple GitHub organizations. This new provider brings in changes to the default plugin configuration settings. Because this is a breaking configuration change, you must upgrade the `@backstage/plugin-catalog-backend-module-github-org` plugin and reconfigure it in the following manner:
```
Comment (hmanwani-rh) [inline release-notes/con-breaking-changes.adoc:8]:
```suggestion
The GitHub Organization catalog module for the new backend system switched to a new `GithubMultiOrgEntityProvider` provider, which allows ingesting data from multiple GitHub organizations. This new provider brings in changes to the default plugin configuration settings. You must upgrade the `@backstage/plugin-catalog-backend-module-github-org` plugin and reconfigure it in the following manner:
```

**Review by hmanwani-rh — COMMENTED** [review]
@shipsing One comment below. Rest looks good.

**Review by hmanwani-rh — APPROVED** [review]

**Pull request merged**